.github/workflows/unit-tests.yml

```diff
@@ -11,6 +11,7 @@ on: # rebuild any PRs and main branch changes
 
 jobs:
    build: # make sure build/ci works properly
+      name: Run Unit Tests
```

Comment:
Same for this file for any action used which is outdated like checkout. ☕️💡

Reply:
good catch, ive updated it now